core/utils/poly-look/babel.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  presets: ["@babel/preset-env", "@babel/preset-react"],
```

Comment:
When #640 lands, the first one can be eliminated; actually, if the second one is used in a couple of places we might be able to hoist it too.